src/sysroot.rs

```diff
@@ -1,166 +0,0 @@
-//! Download and manage sysroots.
```

Comment:
We use bisect-rust's sysroot management now, but I guess I forgot to actually delete the file. Thanks!